cirq/ion/ion_gates.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Operations native to iontrap systems."""
+
+import numpy as np
+
+from cirq import ops
+
+
+def MSGate(rads: float) -> ops.XXPowGate:
+    """
+    Implements the Mølmer–Sørensen gate,
+    native two-qubit operation in ion traps.
+    Effectively equivalents to rotation around the XX axis
+    in the two-qubit bloch sphere.
+
+    The gate implements the following unitary:
+    exp(-i t XX) = [cos(t) 0 0 -isin(t)]
+                   [0 cos(t) -isin(t) 0]
+                   [0 -isin(t) cos(t) 0]
+                   [-isin(t) 0 0 cos(t)]
+    rads = t, the rotation angle in radians
```

Comment:
Put this in an `Args:` section so that tools parse it.

```
Args:
    rads: The rotation angle in radians.

Returns:
    Mølmer–Sørensen gate rotating by the desired amount.
```